web/templates/left_sidebar.hbs

```diff
@@ -146,8 +146,8 @@
     <div id="direct-messages-sticky-header" class="direct-messages-container zoom-out hidden-for-spectators">
         <div id="direct-messages-section">
             <div id="direct-messages-section-header" class="zoom-out zoom-in-sticky">
-                <i id="toggle-direct-messages-section-icon" class="fa fa-sm fa-caret-down dm-tooltip-target toggle_private_messages_section zoom-in-hide" aria-hidden="true" tabindex="0" role="button"></i>
-                <h4 class="left-sidebar-title toggle_private_messages_section"><span class="dm-tooltip-target">{{t 'DIRECT MESSAGES' }}</span></h4>
+                <i id="toggle-direct-messages-section-icon" class="fa fa-sm fa-caret-down dm-tooltip-target zoom-in-hide" aria-hidden="true" tabindex="0" role="button"></i>
+                <h4 class="left-sidebar-title"><span class="dm-tooltip-target">{{t 'DIRECT MESSAGES' }}</span></h4>
```

Comment:
Confirmed using `git log -Stoggle_private_messages_section` that this was introduced in 6f9e97921d39e87637c654337c77daa809080c7b and never used.